### ISSUE TITLE
Conversion error: Jekyll cannot read .scss files in the /_sass directory using the @import feature

### ISSUE THREAD

**Issue by RinseJ** [juthilo/run-jekyll-on-windows#48]
Hello Juthilo,

I successfully installed jekyll for windows a few days ago but every time i try to run the 'jekyll serve' command I always get the same error whatever I try (see below). The error always to exist for the first line in the first file that jekyll encounters for conversion that uses the '@import' feature to import .scss files from the general /_sass directory.

![conversion error](https://cloud.githubusercontent.com/assets/17257482/13058660/eba30db0-d424-11e5-84c3-c8f5f1ab1502.png)

In the example I provided I used the 'feeling responsive' theme but the problems are identical for every project I try including the standard theme jekyll comes with pre-installed.

The only workaround for the problem I have found yet is to copy paste all of the contents from the /_sass .scss files into the main.scss file in /css folder without using the @import feature. Doing so gave the the desired result = and a working web page on server localhost:4000, so the rest of the program seems to be working normally. However doing this for every project would be highly impractical.

I hope that i was clear enough in my description of the problem. I could not find a working solution for my problem anywhere else on the internet so I thought this was probably the best place to ask for one. I hope you or anyone else could provide me with an answer.

Thank you very much!

**Comment by RinseJ** [comment]
Well, I just found the solution for my own problem.

Apparently the error/bug is caused by the name I gave the project folder. Note how in my previous example-image I named my folder the long descriptive name: ResponsiveAuréus. Now look at the next image:

![conversion error solved](https://cloud.githubusercontent.com/assets/17257482/13087359/bd0072ae-d4e7-11e5-9578-bcde5896d9a3.png)

Both maps host exactly the same files (the standard files that are generated for any new folder made using the command 'jekyll new ...'), the only difference lies the name of the folder! the first folder gives an error while the other folder does not.

So, in short, for future documentation: In order to prevent an @import conversion error in your personal jekyll project it is strongly advised to not use an accent aigu in your project folder name.
